streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -595,7 +595,8 @@ public class StreamsConfig extends AbstractConfig {
     public static final String DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS = "default.windowed.value.serde.inner";
     private static final String DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS_DOC = "Default serializer / deserializer for the inner class of a windowed value. Must implement the " +
         "<code>org.apache.kafka.common.serialization.Serde</code> interface.";
-
+    
+    @Deprecated
```

Comment:
We should add JavaDocs similar to `window.size.ms`.

Reply:
yes. done.